Add documentation for creating project superuser (#106)
18F/federal-grant-reporting: README.md

The app should now be running at http://localhost:8000 or http://127.0.0.1:8000.

### Creating a superuser

The Django project contains an admin interface at `/admin`.

To access the admin interface, you'll need to create a superuser first. Run the following on your command line to create a superuser on your local machine:

```
./manage.py createsuperuser --settings=single_audit.settings.development
```

You will be prompted to supply a username, email, and password.

> Once your superuser is set up, you will be able to access the admin interface at `/admin` using your new superuser account.

Reviewer: ```suggestion
Having completed that process, you will be able to access the admin interface at `/admin` using your new superuser account.
```
